File: examples/time/now/node/returnsCurrentTimeOptionallyWithLocation.js
const { TimeService } = require("m3o/time");

// Get the current time
async function returnsCurrentTimeOptionallyWithLocation() {
  let timeService = new TimeService(process.env.MICRO_API_TOKEN);
  let rsp = await timeService.now({});
  console.log(rsp);
}

returnsCurrentTimeOptionallyWithLocation();

File: examples/go/time/now/returnsCurrentTimeOptionallyWithLocation.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/time"
)

// Get the current time
func ReturnsCurrentTimeOptionallyWithLocation() {
	timeService := time.NewTimeService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := timeService.Now(&time.NowRequest{})
	fmt.Println(rsp, err)
}
